### ISSUE TITLE
Polling Error

### ISSUE THREAD

**Issue by elit3ge** [FoxxMD/multi-scrobbler#88]
[Please check the FAQ](https://github.com/FoxxMD/multi-scrobbler/blob/master/docsite/docs/FAQ.md) before submitting a bug report.

**Describe the bug**
After booting Multi-Scrobbler, it polls for a little while and then errors and stops.

**To Reproduce**
Start up multi-scrobbler and wait.

**Expected behavior**
It to work and keep polling.

**Logs**
If possible reproduce the issue with [debug logging ON](https://github.com/FoxxMD/multi-scrobbler/blob/master/docsite/docs/FAQ.md#turn-on-debug-logging)

```
2023-07-22T13:00:14+10:00 debug   : [Sources] [Lastfm - myLastFM] No new tracks discovered
2023-07-22T13:00:14+10:00 debug   : [Sources] [Lastfm - myLastFM] Last activity was at 2023-07-22T12:49:01+10:00 which is 6 minutes outside of active polling period of (last activity + 300 seconds). Will sleep for max interval 60 seconds.
2023-07-22T13:00:14+10:00 verbose : [Sources] [Lastfm - myLastFM] Sleeping for 60s
2023-07-22T13:01:14+10:00 debug   : [Sources] [Lastfm - myLastFM] Refreshing recently played
2023-07-22T13:01:19+10:00 error   : [Sources] [Lastfm - myLastFM] Error occurred while polling
2023-07-22T13:01:19+10:00 error   : [Sources] [Lastfm - myLastFM] getaddrinfo EAI_AGAIN ws.audioscrobbler.com {"code":"EAI_AGAIN","errno":-3001,"hostname":"ws.audioscrobbler.com","syscall":"getaddrinfo"}
Error: 

    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:107:26)

2023-07-22T13:01:19+10:00 warn    : [Sources] [Lastfm - myLastFM] Poll retries (0) equal to max poll retries (0), stopping polling!
```

**Versions (please complete the following information):**
Provide version information for any related sources/clients.

- multi-scrobbler: foxxmd/multi-scrobbler:develop
- maloja: 3.1.4

**Additional context**
Used to work, I have polling at 0.

**Comment by FoxxMD** [comment]
Hi can you try using this docker image and see if this resolves your issue:

```
foxxmd/multi-scrobbler:experimental
```

**Comment by elit3ge** [comment]
New error.

```
2023-07-25T20:06:18+10:00 error   : [Sources] [Lastfm - myLastFM] getaddrinfo EAI_AGAIN ws.audioscrobbler.com {"code":"EAI_AGAIN","errno":-3001,"hostname":"ws.audioscrobbler.com","syscall":"getaddrinfo"}

Error: 

    at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:108:26)

2023-07-25T20:06:18+10:00 warn    : [Sources] [Lastfm - myLastFM] Poll retries (0) equal to max poll retries (0), stopping polling!```

**Comment by FoxxMD** [comment]
Have you made any changes to your network recently? Anything like a new router, setting up pihole or changing dns at the docker, host, or router level?

I can't reproduce your issue and haven't had any other users report this. The error you receiving is due to a DNS server not responding which is very likely outside of the control of multi-scrobbler https://github.com/nodejs/node/issues/15780#issuecomment-334496907

If you are using a [file-based configuration](https://foxxmd.github.io/multi-scrobbler/docs/configuration#file-based-configuration) (`config.json` or `lastfm.json` as opposed to ENV) you can increase the number of polling/retry attempts -- by specifying `maxPollRetries` or `maxRequestRetries` in the `data` property -- and see if that solves things for you:
 
`lastfm.json` example:

```
{
  "clients": [
    "MyMalojaConfig",
  ],
  "data": {
    "clientId": "787c921a2a2ab42320831aba0c8f2fc2",
    "clientSecret": "ec42e09d5ae0ee0f0816ca151008412a",
    "redirectUri": "http://localhost:9078/callback",
    "maxPollRetries": 3,
    "maxRequestRetries": 2
  },
  "name": "MyLastFmClient"
}
```

**Comment by FoxxMD** [comment]
The DNS fix is included in [0.6.0](https://github.com/FoxxMD/multi-scrobbler/releases/tag/0.6.0)
